Aztec/Classes/TextKit/TextStorage.swift

```diff
@@ -363,6 +366,48 @@ open class TextStorage: NSTextStorage {
 }
 
 
+// MARK: - Header Font Attribute Fixes
+//
+private extension TextStorage {
+
+    /// Ensures that the TextFont matches the new HeaderLevel that's about to be applied.
```

Comment:
A header style may not be applied at all, so the description of this method is a bit off.

Something along the lines of `Ensures the font style is consistent with the paragraph header style` would be closer to the truth.